content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,473 @@
+# How to build a task manager with angular.
+
+### Introduction
+As developers, it is common practice to build simple applications. This will help solidify one's knowledge about a particular programming language or framework. 
+
+In this article, we will build a simple task manager application (AKA To-do app) using [Angular](https://angular.io/docs). Angular is a JavaScript framework.
```

Comment:
Could you rewrite this sentence? I feel like 'Angular is a JavaScript framework.' doesn't fit here after giving reader(s) the link.